Copy updates for the Optimized Checkout (#4558)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 9.8.0 - xxxx-xx-xx =

> * Update - Copy for the Optimized Checkout setting to reflect Stripe suggestions

Reviewer: I think we can simplify this to drop the Stripe mention.
```suggestion
* Update - Copy for the Optimized Checkout settings and notices
```
